client/localization/elements/sc-page-dictionary/pl.json

```diff
@@ -3,6 +3,6 @@
     "definitionsFor": "Definicje dla",
     "relatedItems": "Powiązane Przedmioty",
     "adjacentTerms": "Sąsiednie Terminy",
-    "similarTerms": "Podobne Terminy"
+    "similarSpelling": "Podobne Pisownia"
```

Comment:
should be "Podobna Pisownia", as it's a feminine noun :)

Reply:
:-) Will do!